website/source/docs/connect/proxies/envoy.md

```diff
@@ -18,14 +18,18 @@ Consul can configure Envoy sidecars to proxy http/1.1, http2 or gRPC traffic at
 L7 or any other tcp-based protocol at L4. Prior to Consul 1.5.0 Envoy proxies
 could only proxy tcp at L4.
 
-Currently configuration of additional L7 features is limited, however we have
-plans to support a wider range of features in the next major release
-cycle.
-
-As an interim solution, you can add [custom Envoy configuration](#custom-configuration)
+As an interim solution, you can add [custom Envoy configuration](#advanced-configuration)
```

Comment:
"interim solution" is a bit confusing now as we've removed the statement about the current state not supporting L7. Maybe just remove this line and rely on the part below?

Reply:
I actually meant to remove this but did not. Fixed.